qt/scientific_interfaces/Indirect/IndirectPlotOptionsPresenter.h

```diff
@@ -15,38 +15,45 @@
 
 #include <Poco/NObserver.h>
 
-#include <QObject>
-
 namespace MantidQt {
 namespace CustomInterfaces {
 
-class MANTIDQT_INDIRECT_DLL IndirectPlotOptionsPresenter : public QObject {
-  Q_OBJECT
+class MANTIDQT_INDIRECT_DLL IIndirectPlotOptionsPresenter {
+public:
+  virtual void notifyWorkspaceChanged(std::string const &workspaceName) = 0;
+  virtual void notifySelectedUnitChanged(std::string const &unit) = 0;
+  virtual void notifySelectedIndicesChanged(std::string const &indices) = 0;
+  virtual void notifyPlotSpectraClicked() = 0;
+  virtual void notifyPlotBinsClicked() = 0;
+  virtual void notifyPlotContourClicked() = 0;
+  virtual void notifyPlotTiledClicked() = 0;
```

Comment:
Since this is an implementation of observer pattern, isn't is better to have names like **update**WorkspaceChanged rather than **notify**WorkspaceChanged on the observer/subscriber side(which is the Presenter). On the View side(the observable) its fine to have the methods like notifySelectedWorkspaceChanged.